Building DAG of jobs...
Using shell: /bin/bash
Provided cores: 2
Rules claiming more threads will be scaled down.
Job stats:
job          count    min threads    max threads
---------  -------  -------------  -------------
all              1              1              1
df_sirius        1              1              1
sirius           1              1              1
total            3              1              1

Select jobs to execute...

[Sat Sep 11 11:21:05 2021]
rule sirius:
    input: results/preprocessed/Epemicins.featureXML, results/precursorcorrection/Epemicins.mzML, resources/sirius, data/mzML/Epemicins.mzML
    output: results/SIRIUS/Epemicins.mzTab
    jobid: 1
    wildcards: samples=Epemicins
    resources: tmpdir=/var/folders/c_/ysz9v_bd1yb7h3ymmkn6m199jbv7x7/T

[Sat Sep 11 11:21:05 2021]
Error in rule sirius:
    jobid: 1
    output: results/SIRIUS/Epemicins.mzTab
    conda-env: /Users/eeko/Desktop/Metabolomics/Snakemake/.snakemake/conda/66a0911470347801c428959a60e2d219

RuleException:
URLError in line 12 of /Users/eeko/Desktop/Metabolomics/Snakemake/workflow/rules/sirius.smk:
<urlopen error [Errno 2] No such file or directory: '/Users/eeko/Desktop/Metabolomics/Snakemake/workflow/rules/scripts/4_sirius.py'>
  File "/Users/eeko/Desktop/Metabolomics/Snakemake/workflow/rules/sirius.smk", line 12, in __rule_sirius
  File "/Users/eeko/mambaforge/envs/snakemake/lib/python3.9/urllib/request.py", line 214, in urlopen
  File "/Users/eeko/mambaforge/envs/snakemake/lib/python3.9/urllib/request.py", line 517, in open
  File "/Users/eeko/mambaforge/envs/snakemake/lib/python3.9/urllib/request.py", line 534, in _open
  File "/Users/eeko/mambaforge/envs/snakemake/lib/python3.9/urllib/request.py", line 494, in _call_chain
  File "/Users/eeko/mambaforge/envs/snakemake/lib/python3.9/urllib/request.py", line 1481, in file_open
  File "/Users/eeko/mambaforge/envs/snakemake/lib/python3.9/urllib/request.py", line 1520, in open_local_file
  File "/Users/eeko/mambaforge/envs/snakemake/lib/python3.9/concurrent/futures/thread.py", line 52, in run
Shutting down, this might take some time.
Exiting because a job execution failed. Look above for error message
Complete log: /Users/eeko/Desktop/Metabolomics/Snakemake/.snakemake/log/2021-09-11T112104.835258.snakemake.log
